samples/core/src/test/resources/config/application.properties

```diff
@@ -1,10 +1,15 @@
 # https://docs.spring.io/spring-boot/docs/current/reference/html/howto-embedded-servlet-containers.html#howto-user-a-random-unassigned-http-port
 server.port=0
+#Comment the following line in the case of database other than in-memory database H2
 spring.profiles.active=junit
 
+#UnComment the following line in the case of Oracle 11g database
+#spring.profiles.active=junit,orcl
```

Comment:
Good! Programmers will notice that it is important to define all necessary profiles!